### PR TITLE
Rails: Forms and Authentication

### PR DESCRIPTION
**Pull request by krys2fa**
### Project: Members only

In this project, 
 - we created an exclusive a rails application where members can write posts about non-members.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    <div class="column is-4">
+<h1>Post</h1>
+
+<% @posts.each do |p| %>
+  <div class="content">
+    <% if user_signed_in? %>
+      <strong><%= p.user.name %></strong><br />
+    <% end %>
+    <small><%= p.title %></small><br/>
```
Comment (mcrd25) [inline app/views/posts/index.html.erb]: I should also see the body

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Changes Required 🔍 

Hi Team 👋 

You are doing a good job! In fact, you've met all the functional requirements! However there is one minor issue in your execution I could not overlook.

In posts/new route It is not clear which box the post is to be entered. Please fix it. the layout is really weird. I'd have expected the post to be the `textarea` field which I believe it is! 
Screen Shot 2020-06-22 at 2 16 49 PM

However, when I go to the home route I do not see the body of the post I only see the title. Please address this.

Other than this. Good job! 👏 

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by stratospherique — APPROVED** [review]
### Great work implementing the suggested changes :clap: :clap: Congrats, the project is approved :heavy_check_mark: You may merge :+1: :+1: 

Happy coding :v: 
**_Ahmed Mahfoudh_**

**Comment by krys2fa** [comment]
Thanks @stratospherique.

**Pull request merged**